docusaurus/docs/learn/quickstarts/network/miniziti.bash

```diff
@@ -89,24 +95,35 @@ detectOs(){
     fi
 }
 
-getClientOttPath(){
-    local ID_PATH="/tmp/${MINIKUBE_PROFILE}-client.jwt"
-    if [[ $# -eq 1 ]]; then
-        case $1 in
-            macOS|Linux)     echo "$ID_PATH"
+makeMinizitiStateDir() {
+    local detected_os="$1"
+
+    case "$detected_os" in
+        MacOS)
+            state_dir="${XDG_STATE_DIR:-$HOME/Library/Application Support}/miniziti"
             ;;
-            Windows)         wslpath -w "$ID_PATH"
+        Linux)
+            state_dir="${XDG_STATE_DIR:-$HOME/.local/state}/miniziti"
             ;;
-        esac
-    else
-        logError "getClientOttPath() takes one param, DETECTED_OS"
-        return 1
+        Windows)
+            wsl_state_dir="${XDG_STATE_DIR:-$HOME/.local/state}/miniziti"
+            state_dir="$(wslpath -w "$wsl_state_dir")"
```

Comment:
```suggestion
            windows_state_dir="$(wslpath -w "$state_dir")"
```

We're not using `windows_state_dir` yet, but we might if we decide to resurrect `getClientOttPath()`.